config/manifests/bases/watermarkpodautoscaler.clusterserviceversion.yaml

```diff
@@ -39,128 +39,21 @@ metadata:
         }
       ]
     capabilities: Basic Install
-    operators.operatorframework.io/builder: operator-sdk-v1.5.0
+    operators.operatorframework.io/builder: operator-sdk-v1.23.0
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v2
-  name: watermarkpodautoscaler.v0.3.0-rc.1
+  name: watermarkpodautoscaler.v0.6.0-rc.1
   namespace: placeholder
 spec:
   apiservicedefinitions: {}
-  customresourcedefinitions:
-    owned:
-    - description: WatermarkPodAutoscaler is the Schema for the watermarkpodautoscalers API
-      displayName: Watermark Pod Autoscaler
-      kind: WatermarkPodAutoscaler
-      name: watermarkpodautoscalers.datadoghq.com
-      version: v1alpha1
+  customresourcedefinitions: {}
```

Comment:
it is weird that this file is now almost empty

Reply:
agreed, I can look into this when I cut 0.6.0-rc.X, this is just so I can cut it